Bottom: 60adf5d239cf2a29b2c7d01e248e2ea3c7e76edb
Top:    43fe26892289ed67b67c6d60e767a25ed5d67b41
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 13:31:05 -0500

Refresh of generate-new-ssh-key-on-first

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 68181dcc..e14b93b9 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -12,3 +12,26 @@
     role: buildbot-master
     tags:
     - buildbot-master
+
+  - name: Add buildbot user to Gitlab
+    role: gitlab-user
+    vars:
+      new_user: buildbot
+      new_user_name: Build Robot
+      new_user_email: buildbot@novatech-llc.com
+      new_user_pass: "{{ lookup('password', '/dev/null') }}"
+
+- name: Gitlab public keys for buildbot containers
+  hosts: daedalus
+  become: true
+  become_user: root
+  become_method: sudo
+  tags:
+  - buildbot
+  roles:
+  - name: Add buildbot_master pubkey to buildbot user
+    role: gitlab-sshkey
+    vars:
+      username: buildbot
+      pubkey: "{{ hostvars.buildbot_master.pubkey }}"
+      title: "buildbot_master"
